meson.build

```diff
@@ -0,0 +1,60 @@
+project('rhino-setup',
+        'rust',
+          version: '0.1.0',
+    meson_version: '>= 0.59.0',
+    license: 'GPL-3.0-or-later'
+)
+
+i18n = import('i18n')
+gnome = import('gnome')
+
+base_id = 'org.rhinolinux.RhinoSetup'
+
+dependency('glib-2.0', version: '>= 2.66')
+dependency('gio-2.0', version: '>= 2.66')
+dependency('gtk4', version: '>= 4.0.0')
+dependency('libadwaita-1', version: '>= 1.0.0')
+
+glib_compile_resources = find_program('glib-compile-resources', required: true)
+glib_compile_schemas = find_program('glib-compile-schemas', required: true)
+desktop_file_validate = find_program('desktop-file-validate', required: false)
+# appstream_util = find_program('appstream-util', required: false)
+cargo = find_program('cargo', required: true)
+find_program('gettext', required: true)
```

Comment:
This shouldn't be needed. Meson checks internally for msgfmt, msgmerge, etc. -- but `gettext "message string"` is a bash scripting command to print the translated version, and is not actually required for a successful build.